app/views/pages/release-notes/2020/MAY/25.liquid

```diff
@@ -0,0 +1,36 @@
+---
+converter: markdown
+metadata:
+  title:
+  description: May 25, 2020
+  skip_contribute_button: true
+  last_edit: false
+slug: release-notes/25-MAY-2020
+---
+
+#### {{ page.metadata.description }}
+
+<h4 class="release-note release-note__new">NEW</h4>
+
+* Added `request_allowed` flag to Form to control whether a form will be available as an endpoint (which is deprecated). If `false`, the form will be usable only through GraphQL mutation. Default is `true`.
+
+<h4 class="release-note release-note__improved">IMPROVED</h4>
+
+* error message for not sending Email Notification
+* added more examples to extract_url_params and link to the official RFC
+* `to_mobile_number` filter will add `+` if the phone starts with country code
+* `map` filter implementation and its description
+* added case insensitive modifier and multiline option to `regex_matches` and `matches` filters
+* *DEPRECATION* rename `headers` to `request_headers` in ApiCalls to avoid ambiguity and be consistent with `response_headers` in Page
```

Comment:
```suggestion
* *DEPRECATION* rename `headers` to `request_headers` in api calls notifications to avoid ambiguity and be consistent with `response_headers` in Pages
```